jerry-port/default/default-date.c

```diff
@@ -98,20 +97,7 @@ double jerry_port_get_local_time_zone_adjustment (double unix_ms,  /**< ms since
  */
 double jerry_port_get_current_time (void)
 {
-#ifdef __GNUC__
-  struct timeval tv;
-
-  if (gettimeofday (&tv, NULL) == 0)
-  {
-    return ((double) tv.tv_sec) * 1000.0 + ((double) tv.tv_usec) / 1000.0;
-  }
-#endif /* __GNUC__ */
-
-#ifdef _WINDOWS
   time_t ltime;
   time (&ltime);
-  return ltime * 1000;
-#endif /* _WINDOWS */
-
-  return 0.0;
+  return (double) ltime * 1000.0;
```

Comment:
But `time()` has seconds granularity only! `jerry_port_get_current_time` is used by `Date.now()`, which is returns the number of _milliseconds_ elapsed since January 1, 1970 00:00:00 UTC. I think the Windows-specific code path used `time()` only as a fallback, as (non-MinGW) Windows has no `gettimeofday()`. But I don't think that we should lower precision generally.

Reply:
Ah, I missed it. In this case, I think we should keep my original proposal with pragmas not to lower precision for MinGW build.

Reply:
Or maybe we can get epoch time on Windows with ms granularity. Let me try, what I have found.

Reply:
I found the best solution, we should use GetSystemTimeAsFileTime() on Windows. It's a little bit tricky,
because we have to convert Windows Epoch (1601) to Unix Epoch (1970) and the granularity is different too.